sound/soc/sof/ipc4-topology.c

```diff
@@ -1129,7 +1138,7 @@ sof_ipc4_prepare_copier_module(struct snd_sof_widget *swidget,
 			if (blob->alh_cfg.count > 1) {
 				copier_data->gtw_cfg.node_id &= ~SOF_IPC4_NODE_INDEX_MASK;
 				copier_data->gtw_cfg.node_id |=
-					SOF_IPC4_NODE_INDEX(ALH_MULTI_GTW_BASE);
+					SOF_IPC4_NODE_INDEX(ALH_MULTI_GTW_BASE + node_id_shift);
```

Comment:
@bardliao this logis is incorrect I think. 1. This would always start node index from BASE + 1. 2. Assuming you have more than 1 group for aggreghation, if you start 2 groups, then stop the first group and restart it, what node_index would it get?

Reply:
@bardliao if you used ida_assign/free to assign the group ID at runtime, you wont even need to use the use_count. The question is where to save the ida. im sure you can figure that out

Reply:
> @bardliao this logic is incorrect I think. 1. This would always start node index from BASE + 1. 2. Assuming you have more than 1 group for aggreghation, if you start 2 groups, then stop the first group and restart it, what node_index would it get?

1. No, it won't. the use_count of the widget itself is still 0 at this point. 2. Yeah, that will be a problem. It will be 1, just the same as the second group. Thanks for pointing this out.

Reply:
> @bardliao if you used ida_assign/free to assign the group ID at runtime, you wont even need to use the use_count. The question is where to save the ida. im sure you can figure that out

My first thought is to use ida, but I didn't figure out where should we save the ida. Maybe I should reconsider this solution if not only me think ida is a good idea.